### ISSUE TITLE
Error on share / collboration

### ISSUE THREAD

**Issue by ar090** [PatWie/excalidraw-complete#4]
Thanks for putting this together! Love the idea of self hosting with the binary makes things incredibly simple. 
I am getting this error trying to export to file:
<img width="547" alt="image" src="https://github.com/PatWie/excalidraw-complete/assets/17073553/fa44254b-4425-483b-a961-5083b31d4ba6">

Additionally, collaboration button is giving something similar in the console
<img width="671" alt="image" src="https://github.com/PatWie/excalidraw-complete/assets/17073553/555eacdf-b82c-4994-b2f9-e6bc4203d8f5">

I'm likely missing some configuration setup somewhere. 
Using 0.0.2 for linux -> [excalidraw-complete_Linux_x86_64.tar.gz](https://github.com/PatWie/excalidraw-complete/releases/download/0.0.2/excalidraw-complete_Linux_x86_64.tar.gz) 
I tried without any env variables and also with the file storage ones from the README.

Last question - for the AWS bucket storage, is this live? Guessing the go lib will look for the standard AWS env variables for secret / access keys?

Thanks again!

**Comment by PatWie** [comment]
I am currently on travel. Will have a look at that error when back. 

But yes, all storage options are tested and are working. It does read the default AWS profile credentials

**Comment by PatWie** [comment]
The frontend UI has the urls hard-coded to localhost:3002 -- even when hosting under some different domain/IP. You cannot simply put it on EC2 without rebuilding the UI.

You will need to adapt the git patch matching your setup (domain/url) and then build the UI filtering the readme and then the go source from scratch.

I can assure you it is working on EC2. I do it myself under a custom domain (using Cloudfront/Alb/Fargate).

**Comment by ar090** [comment]
@PatWie this is exactly what I am trying to accomplish. Super helpful thank you! I will take the git patch route.

**Comment by ar090** [comment]
I figured this out If anyone hits a similar snag its coming from this line:
https://github.com/excalidraw/excalidraw/blob/a04676d4234e9f7d33500307b6a0481387820f3a/packages/excalidraw/data/encryption.ts#L16

TLDR Wont work unless your are on HTTPS
